mappings/net/minecraft/util/math/MutableBox.mapping

```diff
@@ -28,27 +28,33 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
 		ARG 1 other
 	METHOD method_14658 toNbt ()Lnet/minecraft/class_2495;
 	METHOD method_14659 getSize ()Lnet/minecraft/class_2382;
```

Comment:
New suggestion: `getSize` -> `getDimensions`. The latter is technically more correct and disambiguates it from the `getXYZSize` methods.

Reply:
So `getXDimention`?

I haven't looked at this for a few days, but iirc `getSize` here returns a scalar value, so perhaps `getVolume` would be better?

Reply:
Nop, keep `getXSize`, but rename the base `getSize` to `getDimensions`. That is because sizes are single values, whereas dimensions are vectors.